### PR TITLE
Task/psgs 57 missing apis migrations

### PR DESCRIPTION
**Pull request by maciejbak85**

### CODE REVIEW

**Comment by maciejbak85** [comment]
re-tested manually today, all good

File: src/test/scala/iog/psg/cardano/util/ResourceFiles.scala [hunk 1]
```diff
@@ -34,7 +36,10 @@ trait ResourceFiles { self: Assertions =>
 
   final def decodeJsonFile[T](file: String)(implicit dec: Decoder[T]): T = {
     val jsonStr = getJsonFromFile(file)
-    decode[T](jsonStr).getOrElse(fail(s"Could not decode $file"))
+    decode[T](jsonStr) match {
```
Comment (mcsherrylabs) [inline src/test/scala/iog/psg/cardano/util/ResourceFiles.scala:39]: This is unnecessary, right?

**Review by mcsherrylabs — APPROVED** [review]
Our coverage is quite low. I assume it's not because of this PR. But it's a concern.

File: src/test/scala/iog/psg/cardano/util/ResourceFiles.scala (hunk 1 above)
Comment (maciejbak85) [reply src/test/scala/iog/psg/cardano/util/ResourceFiles.scala:39]: well its useful because it adds error message to test failure

**Pull request merged**